test/unit/isURL-test.js

```diff
@@ -0,0 +1,26 @@
+const { assert } = require('chai');
+
+const isURL = require('../../lib/isURL');
+
+
+describe('isURL()', () => {
+  it('recognizes HTTP URL', () => {
+    assert.isTrue(isURL('http://example.com'));
+  });
+
+  it('recognizes HTTPS URL', () => {
+    assert.isTrue(isURL('https://example.com'));
```

Comment:
Do we need to assert if the URL is actually valid?
Urls like `https://foo` will also match the RegExp. Would it be enough to error upon trying to request those resources?

Reply:
I don't think it's worth. Dredd tries to fetch the API description from the URL and it can fail for many reasons, and the URL being invalid is just one of them. This is only for the purpose of knowing which locations to try to download and which are local files.

Reply:
That means,

> Would it be enough to error upon trying to request those resources?

Yes 😄